apps/charterafrica/src/components/OpportunityCard/OpportunityCard.js

```diff
@@ -0,0 +1,72 @@
+import { Card, CardContent, CardMedia, Typography } from "@mui/material";
+import React from "react";
+
+import { neutral, secondary } from "@/charterafrica/colors";
+
+const OpportunityCard = React.forwardRef(function OpportunityCard(props, ref) {
+  const { opportunity } = props;
+
+  if (!opportunity) {
+    return null;
+  }
+
+  return (
```

Comment:
```suggestion
  if (!opportunity) {
    return null;
  }
  return (
```
basically if ... else , no need for space in between.